godot-core/src/builtin/meta/godot_compat/impls.rs

```diff
@@ -0,0 +1,248 @@
+/*
```

Comment:
This file could benefit from the `// ----` horizontal delimiters to separate impl groups per type.